Sources/Sentry/SentrySwizzle.m

```diff
@@ -20,6 +20,9 @@ @implementation SentrySwizzleInfo
 - (SentrySwizzleOriginalIMP)getOriginalImplementation
 {
     NSAssert(_impProviderBlock, nil);
```

Comment:
@armcknight mentioned the redundant `nil` here, but if I remove it the code no longer compiles 🤔
`Too few arguments provided to function-like macro invocation`

So I guess the `nil` was added with good reason. Or we could give it a proper string instead?

Reply:
We should add a message, otherwise we get an assertion/exception with an empty message 
<img width="322" alt="image" src="https://user-images.githubusercontent.com/3241469/189726529-01cd3684-ea64-4e2a-8a38-ca454df1ed30.png">

Reply:
Typically in the past I've wrapped this in a macro like `SENTRY_ASSERT_NONNULL`

Reply:
I added a message